system/security/securesocket/securesocket.cpp

```diff
@@ -1268,9 +1268,10 @@ static bool setVerifyCertsPEMBuffer(SSL_CTX *ctx, const char *caCertBuf, int caC
     return true;
 }
 
-class CSecureSocketContext : public CInterfaceOf<ISecureSocketContext>
+class CSecureSocketContext : implements ISecureSocketContext, implements ISecureSocketContextCallback, public CInterface
```

Comment:
use a CInterfaceOf< to remove 1 vmt?

Reply:
I could but the IMPLEMENT_USING gets a bit ugly - not enough instances to worry either way.